components/ca-form.js

```diff
@@ -317,9 +317,7 @@ define([
                         this.populate(JSON.parse(formData));
                     }
                 }
-            }).catch(err => {
-                // eslint-disable-next-line
-                alert(`Schema not found! ${err}`);
+            }).catch({
```

Comment:
We're not supposed to be making any behavioural changes here. Empty catches are particularly bad.